include/ginkgo/core/matrix/csr.hpp

```diff
@@ -224,15 +224,15 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
         void process(const Array<index_type> &mtx_row_ptrs,
                      Array<index_type> *mtx_srow)
         {
-            // if the number of stored elements per row is larger than 1e6 or
+            // if the number of stored elements is larger than 1e6 or
             // the maximum number of stored elements per row is larger than
             // 64, use load_balance otherwise use classical
             const auto num = mtx_row_ptrs.get_num_elems();
```

Comment:
I think it would be more clear if this variable was named `num_rows` and it was set to `mtx_row_ptrs.get_num_elems() -1` . It looks a bit confusing now.

Reply:
That is true, good point, I will change that.

Reply:
I uploaded the change now.